README.md

```diff
@@ -90,6 +90,10 @@ singularity exec --bind $(pwd)/fastq:/data staphb-bbtools-38.96.simg bbduk.sh in
 
 Further documentation can be found at [docs.sylabs.io](https://docs.sylabs.io/guides/3.1/user-guide/cli.html)
 
+## Training
+
+- Training materials (slides & exercises) from the North East Bioinformatics Regional Resource 2023 workshop **Docker for Public Health Bioinformatics** can be located here, within [`training/NE-BRR-docker-for-PH-bioinformatics-May2023/`](training/NE-BRR-docker-for-PH-bioinformatics-May2023/)
+
```

Comment:
Could you add something like
```
- Video training from APHL 2024 workshop **Intermediate Docker Bioinformatics Workshop** can be located in [APHL's e-learning resources](https://learn.aphl.org/learn/course/external/view/elearning/355/intermediate-docker-bioinformatics-workshop)
```

Reply:
Done, here's the rendered version: https://github.com/theiagen/docker-builds/tree/master?tab=readme-ov-file#training